### DISCUSSION TITLE
Error pushing to new repository

### DISCUSSION THREAD

**Discussion by kchodorow** [community/community#23059] [New to GitHub]
<p>I created a new repository on GitHub and wanted to push some existing code to it. I added the origin and pushed and got:</p>
<pre><code class="lang-auto">git push -u origin main
ERROR: Repository not found.
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.
</code></pre>
<p>The repo exists on GitHub, so I checked my SSH key:</p>
<pre><code class="lang-auto">$ ssh -T git@github.com
Hi kchodorow/reimagined-pancake! You've successfully authenticated, but GitHub does not provide shell access.
</code></pre>
<p>The weird thing is that the repository I created isn’t reimagined-pancake, that’s a random repo I forked a while ago. It’s probably why I created that SSH key, so I was thought “Hmm, maybe that key is inextricably linked with that old repo?” So I created a new key, id_ed25519_2, and added it to my GitHub account. Then I tried logging in with that key:</p>
<pre><code class="lang-auto">ssh -i ~/.ssh/id_ed25519_2 -T git@github.com
Hi kchodorow/reimagined-pancake! You've successfully authenticated, but GitHub does not provide shell access.
</code></pre>
<p>Anyone know WTF is happening? I just want to push code to a new repo!</p>

**Comment by airtower-luna** [comment, marked as answer]
<p>Run <code>ssh</code> with <code>-v</code> to see what’s happening, especially which keys are actually used. From the outputs you’ve shown I’d guess that the key was added to that repository as a deploy key. If you want to use the key to authenticate as your account you need to remove it as a deploy key first.</p>

**Comment by kchodorow** [comment]
<p>Thank you! I removed the deploy key and now ssh correctly says <code>Hi kchodorow!</code></p>

**Comment by SexyRottenFish** [comment]
Hi, I have a problem when I try to push "Due to U.S. trade controls law restrictions, this repository has been disabled". Why it happens? I live in Germany and there are no sanctions and that's my first repository

**Reply by airtower-luna** [reply]
That's something you'll have to [contact GitHub support](https://support.github.com/request) about, us other users can't help there. Good luck!

Also, that's a completely different problem and should have been a separate thread. :sweat_smile:
